doc/api/assert.md

```diff
@@ -1063,6 +1063,31 @@ If the values are not strictly equal, an `AssertionError` is thrown with a
 `message` parameter is an instance of an [`Error`][] then it will be thrown
 instead of the `AssertionError`.
 
+## assert.stringMatching(input, match)
+<!-- YAML
+added: v10.11.1
```

Comment:
We don't explicitely reference future versions, use `REPLACEME`.

Reply:
Ok, I didn't know about this rule. I will fix it